actiontext/lib/generators/action_text/install/install_generator.rb

```diff
@@ -66,6 +68,21 @@ def create_migrations
         rails_command "railties:install:migrations FROM=active_storage,action_text", inline: true
       end
 
+      def using_js_runtime?
+        @using_js_runtime ||= Pathname(destination_root).join("package.json").exist?
+      end
+
+      def using_bun?
+        # Cannot assume yarn.lock has been generated yet so we look for
+        # a file known to be generated by the jsbundling-rails gem
+        @using_bun ||= using_js_runtime? && Pathname(destination_root).join("bun.config.js").exist?
```

Comment:
Memoization only helps here if this returns `true`. Is there a big performance gain to memoizing  @terracatta? If not we could remove it to simplify the code.